### DISCUSSION TITLE
default settings for all users

### DISCUSSION THREAD

**Discussion by tomiapage** [sumatrapdfreader/sumatrapdf#3996] [Q&A]
Hello, i need to disable this options for all users , how can i do it ? When new user open sumatra pdf this options are set to true
RememberOpenedFiles = false
RestoreSession = false

**Comment by GitHubRulesOK** [comment]
SumatraPDF is a personall user reader so settings are designed for each user makes their own choices.
There are options for restricting file access in kiosk settings but not designed for single entries as you seem to be asking.

You could switch those options off and file lock that in a portable copy with restrictions.ini accessed by all users, or you could much better educate your users to not switch them back on again.

AFAIK if you pre-seed a settings file with desired options then on first start the pre-set user choices will be respected and used in the initial start up, this would be similar to preserving settings on start-up or up-date.

for example on installation if I place a sumatraPDF-settings.txt file with
```
RememberOpenedFiles = false
RestoreSession = false
```
then on first run it will be off and stay that way (see top right) and persist when other options get added unless the user intentionally changes it.
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/80601f09-6a5c-4b0c-88b6-dfc1ef44b613)

**Comment by tomiapage** [comment]
Not working for me..
But whem i have installed sumatra and delete sumatraPDF-settings.txt from user profile (C:\Users\Test\AppData\Local\SumatraPDF) and create sumatraPDF-settings.txt with options in location sumatrapdf.exe location it should work ? I cant uninstall and install again sumatra :/
Version 3.5.2 64 Bit

**Reply by GitHubRulesOK** [reply]
close SumatraPDF (as if not yet installed) and edit and copy the user settings to just what you want to be respected. 
 per user `%localappdata%\SumatraPDF\SumatraPDF-settings.txt ` or `%userprofile%\AppData\Local\SumatraPDF\SumatraPDF-settings.txt` (can be part of their login profile)
then user open SumatraPDF will only have those settings until a file is opened then it will keep track of both session files and all preferences, until you reset on next event.
